packages/rs-sdk/src/sdk.rs

```diff
@@ -952,17 +962,18 @@ impl SdkBuilder {
     pub fn build(self) -> Result<Sdk, Error> {
         PlatformVersion::set_current(self.version);
 
+        let dapi_client_settings = self.settings.unwrap_or(DEFAULT_REQUEST_SETTINGS);
```

Comment:
I think you need to merge passed setting on top of default setting, not replace. We can deal with settings in a separate PR. Let's discuss it and create a task.